File: files/CH-221.py
import pandas as pd
import re

path = "CH-221 Combining the columns.xlsx"
input_data = pd.read_excel(path, usecols="B", skiprows=1, nrows=6)
test = pd.read_excel(path, usecols="E", skiprows=1, nrows=6).rename(columns=lambda col: col.replace('.1', ''))

input_data['Text'] = input_data['Text'].str.replace(r'[a-z]', '', regex=True)

print(input_data.equals(test)) # True

File: files/200-299/221/CH-221.R
library(tidyverse)
library(readxl)

path = "files/CH-221 Combining the columns.xlsx"
input = read_excel(path, range = "B2:B7")
test = read_excel(path, range = "E2:E7")

result = input %>%
  mutate(Text = str_remove_all(Text, "[a-z]"))

all.equal(result, test)
#> [1] TRUE
